index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Paul Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+
+    <link 
+        href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+        rel="stylesheet"
+    />
+</head>
+<body>
+    <header>
+        <nav class="container">
+            <a href="#" class="logo">Paul</a>
+            <i class="material-icons">menu</i>
+        </nav>
+    </header>
+
+    <main>
+      <section class="section">
+      
+        <h1 class="main-title">
+            Hey There. <br>
+            I'm Paul</h1>
+        <h4 class="sub-title">I am a Software Developer</p>
+
+        <p class="description">
+             I can help you build a product, feature or <br>
+             website. Look through some of my work <br>
+             and experience! If you like what you see <br>
+             and have a project you need coded, <br> 
+             don't hestiate to contact me.
+        </p>
+
+        <div class="social-icons">
+          <ul class="social-wrapper">
+            <li>
+              <a href="#">
+                <img src="./images/github.png" alt="" class="social">
+              </a>
+            </li>
+  
+            <li>
+              <a href="#">
+                <img src="./images/linkedin.png" alt="" class="social">
+              </a>
+            </li>
+  
+            <li>
+              <a href="#">
+                <img src="./images/Angellist.png" alt="" class="social">
+              </a>
+            </li>
+  
+            <li>
+              <a href="#">
+                <img src="./images/twitter.png" alt="" class="social">
+              </a>
+            </li>
```

Comment:
As mentioned by the previous reviewer some of the icons are not being loaded properly, and it seems you misspelled their names. For example, the file `Angellist.png` is stored as `angellist.png` in the images directory. In other words, the letter `A` must not be capitalized.

![image](https://user-images.githubusercontent.com/34779928/198728349-a2244215-1b84-49ee-a18e-145def6b3167.png)